models/userSchema.js

```diff
@@ -0,0 +1,35 @@
+import mongoose from "mongoose";
+
+const userSchema = mongoose.Schema(
+  {
+    username: {
+      type: String,
+    },
+    email: {
+      type: String,
```

Comment:
add basic regex for validation
`match: [
			/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/,
			"Please add a valid email",
		],`